src/lib/helpers/null-checker.ts

```diff
@@ -0,0 +1,4 @@
+export function isNullOrUndefined(object:Object):boolean
```

Comment:
```suggestion
export function isNullOrUndefined(object:unknown):boolean
```

`Object` ist nicht der korrekte Typ, da man auch `string` oder `number` mit der Funktion überprüfen kann. Mit `unknown` deckt man alle andere Typen ab und an dieser Stelle spielt der Typ auch keine Rolle 😄 

Ggf. könnte man den Parameter noch zu `value` oder so umbenennen